tun/tun_windows.go

```diff
@@ -160,8 +160,8 @@ retry:
 		packet, err := tun.session.ReceivePacket()
 		switch err {
 		case nil:
-			packetSize := len(packet)
-			copy(bufs[0][offset:], packet)
+			packetSize := min(len(packet), len(bufs[0][offset:]))
+			copy(bufs[0][offset:], packet[:packetSize])
```

Comment:
use the copy return result instead and let it do it?

Reply:
`packetSize := copy(bufs[0][offset:], packet)` ?

Reply:
Updated